### ISSUE TITLE
Copy changes to command help output

### ISSUE THREAD

**Issue by benarmston** [alces-flight/flight-asset-cli#20]
Make the following changes to the help output.

 - Change summary for edit-asset-info command from `flight-asset
   edit-asset-info Asset` to `flight-asset edit-asset-info ASSET`
 - Change SUPPORT_TYPE to TYPE
 - Change INFO_PATH to FILE
 - Change ASSET_GROUP to GROUP, e.g.: `flight-asset recommission-group ASSET_GROUP` becomes `flight-asset recommission-group GROUP`.
 - Correct flight-asset update-asset help text (allows modification of info as well):
```
  SYNOPSIS:

    flight-asset update-asset ASSET

  DESCRIPTION:

    Modify the support type for an asset
```

Also show possible values of support type in the help output.  (This may not be simple and is perhaps suitable to be pulled out into its own issue).

**Comment by WilliamMcCumstie** [comment]
There appears to be a few issues surrounding `support_type`. Investigating

**Comment by WilliamMcCumstie** [comment]
The final issue with `support_type` has been resolved in: 4fbd16b58d8db1f14ee6d11572d5be66ee7a21a8
